src/Hosting.Services.Remoting/HostBuilderExtensions.cs

```diff
@@ -34,10 +41,17 @@ public static class HostBuilderExtensions
 			this ServiceFabricHostBuilder<OmexStatelessService, StatelessServiceContext> builder,
 			string name,
 			FabricTransportRemotingListenerSettings? settings = null)
-				where TService : class, IService =>
-					builder
-						.ConfigureServices((_, services) => services.TryAddTransient<TService>())
-						.AddRemotingListener(name, (provider, _) => provider.GetRequiredService<TService>(), settings);
+				where TService : class, IService
+		{
+			if (settings == null)
+			{
+				settings = new FabricTransportRemotingListenerSettings();
+			}
+			settings.ExceptionSerializationTechnique = FabricTransportRemotingListenerSettings.ExceptionSerialization.Default;
+			return builder
+				.ConfigureServices((_, services) => services.TryAddTransient<TService>())
+				.AddRemotingListener(name, (provider, _) => provider.GetRequiredService<TService>(), settings);
+		}
```

Comment:
Why is the remoting listener creation on lines 59 and 67 unaffected?

Reply:
I am not sure how to add it for them because it is different type of listener function. Can you give me an example for one of them?

Reply:
https://github.com/microsoft/Omex/blob/main/src/Hosting.Services.Remoting/RemotingListenerBuilder.cs could be updated to just set ExceptionSerializationTechnique by default, then no change would be required here.

Reply:
If I am reading the rest of the code here correctly the rest of the calls to .AddRemotingListener in this class end up calling those two extensions. If so, making the change in RemotingListenerBuilder would negate the need to make any changes in this class.